src/ucp/api/ucp.h

```diff
@@ -2409,13 +2409,14 @@ ucs_status_ptr_t ucp_am_send_nb(ucp_ep_h ep, uint16_t id,
  * @ingroup UCP_COMM
  * @brief Send Active Message.
  *
- * This routine sends an Active Message to an ep. If the operation completes
- * immediately and UCP_OP_ATTR_FLAG_NO_IMM_CMPL flag is not set in op_attr_mask
- * field of @a param, the routine returns NULL and the call-back routine is
- * not invoked even if specified. Otherwise, if no error is reported and
- * UCP_OP_ATTR_FIELD_CALLBACK flag is passed in op_attr_mask field of @a param,
- * then the UCP library will schedule invocation of the call-back routine
- * @a param->cb.send upon completion of the operation.
+ * This routine sends an Active Message to an ep. If the operation returns
```

Comment:
need to revert "returns" to "completes" : the operation always returns immediately. sometimes, however, not completes immediately.
@tonycurtis